lib/parsers/index.ts

```diff
@@ -92,3 +94,16 @@ export function createPkgTreeFromDep(dep: Dep): PkgTree {
 
   return pkgTree;
 }
+
+export function getYarnWorkspaces(targetFile: string): string [] | false {
+  try {
+    const packageJson: ManifestFile = parseManifestFile(targetFile);
+    if (!!packageJson.workspaces && !!packageJson.private) {
+      return packageJson.workspaces;
```

Comment:
Would you mind to `return ...packageJson.workspaces;`? By that, you will copy the values from the original object rather than keeping a reference to it. So we will prevent unlikely, but possible issues with weird missing data.